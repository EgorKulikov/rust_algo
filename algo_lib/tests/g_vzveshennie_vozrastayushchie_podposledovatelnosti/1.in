4
5
6 4 8 6 5
4
1 2 3 4
3
3 2 2
4
4 5 6 5
