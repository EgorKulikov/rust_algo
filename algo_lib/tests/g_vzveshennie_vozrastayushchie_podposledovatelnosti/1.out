4
12
0
6
